src/components/sections/Incentives/incentives.scss

```diff
@@ -1,46 +1,43 @@
 @import "src/styles/scaffold";
 
-.incentives {
+[data-fs-incentives] {
+  // --------------------------------------------------------
+  // Design Tokens for Incentives
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-incentives-bkg-color            : var(--fs-color-primary-bkg-light);
+  --fs-incentives-padding-top          : var(--fs-spacing-4);
+  --fs-incentives-padding-bottom       : var(--fs-incentives-padding-top);
+  --fs-incentives-border-color         : var(--fs-border-color-light);
+  --fs-incentives-border-width         : var(--fs-border-width);
+
+  // Title
+  --fs-incentives-title-size           : var(--fs-text-size-1);
+  --fs-incentives-title-weight         : var(--fs-text-weight-bold);
+  --fs-incentives-title-line-height    : 1.42;
+
+  // Description
+  --fs-incentives-description-size          : var(--fs-incentives-title-size);
+  --fs-incentives-description-line-height   : 1.14;
```

Comment:
@renatamottam and @hellofanny, are you planning to have tokens for the line-height? It would be nice to standardize some values in tokens so they don't feel arbitrary.

Reply:
I don't think so. We have associate line-height for titles on Typography, not sure if this works for local values. But might be a good idea.